File: backend/Models/Match.cs
namespace backend.Models
{
    public class Match
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public string Location { get; set; } = string.Empty;
        public int TeamId { get; set; }
        public string Opponent { get; set; } = string.Empty;
        public string Result { get; set; } = string.Empty;
    }
}

File: frontend/types/match.ts
export type MatchListItemDto = {
  id: number;
  date: string;      // ISO
  location: string;
  teamId: number;
  name: string;
  opponent: string;
  result: string;    // "" alebo "2:1"
};

export type MatchesMode = "all" | "upcoming";

export type MatchDetailDto = {
  id: number;
  date: string;
  location: string;
  name: string;
  opponent: string;
  result: string;
  isCoachOfTeam: boolean;
};
export type MatchUpdateDto = {
  date: string;
  location: string;
  opponent: string;
  result: string;
};

